src/Gateways/AbstractGateway.php

```diff
@@ -866,6 +866,14 @@ public function process_refund( $order_id, $amount = null, $reason = '' ) {
 
 		$order        = new WC_Order( $order_id );
 		$request      = $this->prepare_request( $txn_type, $order );
+
```

Comment:
The validation looks good, but let's format it to respect the WooCommerce standards